nmrpystar/test/testparser.py

```diff
@@ -40,7 +40,7 @@ class TestParserErrors(u.TestCase):
     def test_cst_problem(self):
         self.assertEqual(parse_nmrstar_ast("data_hi save_me # oop \n "), 
                          bad({'phase': 'CST construction', 
-                              'message': [('data', (1,1)), ('save', (1,9)), ('save close', 'EOF')]}))
+                              'message': [('data', 'EOF'), ('save', 'EOF'), ('save close', 'EOF')]}))
```

Comment:
@mattfenwick this was possibly not a valid move

Reply:
ooh yeah off the top of my head I'm not sure if this makes sense or not, overall I'm fine with it if it's consistent with everything else and passes the tests, but maybe it should be split into a separate PR?